AOSSIE-Org/Resonate
# [BugFix] Fix for Null Check Operator Used on a Null Value Exception in UiSizes Class

## Description

This PR addresses the issue where a `Null check operator used on a null value` exception is thrown when trying to access `Get.width` and `Get.height` in the `UiSizes` class before the `BuildContext` is available.

The solution involves refactoring the `UiSizes` class to use `late` keyword for the size variables and initializing them in a new `init` method which is called after the `BuildContext` is available. 'MediaQuery' is used to get the screen sizes.
Fixes #288 

## Changes:
1. Refactored the `UiSizes` class to use `late` keyword for the size variables.
2.  Added an `init` method in `UiSizes` class to initialize the size variables using `MediaQuery`.
3. Called `UiSizes.init()` in the `build` method of the main widget.

## How Has This Been Tested?

1. Run the app to ensure it loads without throwing the `Null check operator used on a null value` exception.
2. Check various parts of the app to ensure that the sizes are being applied correctly.


 

https://github.com/AOSSIE-Org/Resonate/assets/144162711/57346fcb-7e5e-4e6b-9fd5-ed527b031c0a
![Screenshot (182)](https://github.com/AOSSIE-Org/Resonate/assets/144162711/1922b727-584a-470e-a4cb-a20029a52a6b)


## Checklist:

- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

## Maintainer Checklist
